components/fraudlabs_pro/actions/verify-otp/verify-otp.mjs

```diff
@@ -0,0 +1,59 @@
+import  * as flpnodejs from 'fraudlabspro-nodejs';
+
+export default {
+  name: "Get Verification Result",
+  description: "Verify that an OTP sent by the Send SMS Verification API is valid. Please refer to the [documentation](https://www.fraudlabspro.com/developer/api/get-result) for the explanation of the result returned.",
+  key: "fraudlabs_pro-verify-otp",
+  version: "0.0.1",
+  type: "action",
+  props: {
+    flp_api_key: {
+      type: "app",
+      app: "fraudlabs_pro",
+      description: "API license key. You can sign up for a trial key at [here](https://www.fraudlabspro.com/subscribe?id=1).",
+    },
+    tran_id: {
```

Comment:
please rename this property in a lowerCase style in that way we are consistent with rest of  components.
```suggestion
    tranId: {
```